functions/utils/getToken.ts

```diff
@@ -0,0 +1,22 @@
+import { TokenDocument } from '../../src/graphql/data/__generated__/types-and-hooks'
+import client from '../client'
+
+export default async function getToken(networkName: string, tokenAddress: string, url: string) {
+  const { data } = await client.query({
+    query: TokenDocument,
+    variables: {
+      chain: networkName,
+      address: tokenAddress,
+    },
+  })
+  const asset = data?.token
+  if (!asset) {
+    return undefined
+  }
+  const formattedAsset = {
+    title: 'Get ' + asset.symbol + ' on Uniswap',
```

Comment:
is symbol always defined?

Reply:
It should be as symbol is part of the constructor for ERC20 contracts, but then again someone can always deployed a modified version so I guess not always.